### PR TITLE
[MLX-41] Pyswitch API for ping utility

### PR DESCRIPTION
**Pull request by dpasupathi**
This PR has changes to add API support at pyswitch wrapper level for ping utility. Inline with NE pack PR:
https://github.com/StackStorm/network_essentials/pull/199

### CODE REVIEW

File: pyswitch/snmp/base/utils.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        for key, value in cli_output.iteritems():
+            output_dict = {}
+            if ipv4_address.search(key):
+                ip = ipv4_address.search(key).group()
+            elif ipv6_address.search(key):
+                ip = ipv6_address.search(key).group()
+            value_list = value.splitlines()
+            success_rate = p_tx = p_rx = 0
+            p_loss = 100
+            for i, line in enumerate(value_list):
```
Comment (vijendarp) [inline pyswitch/snmp/base/utils.py]: where is i used?

File: pyswitch/snmp/mlx/base/utils.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            auth: authentication parameter for device ssh (username, password)
+
+        Returns:
+
+        Raises:
+            None
+        """
+        super(Utils, self).__init__(callback)
+
+    def _create_ping_cmd(self, targets, vrf, count, timeout_value, size):
```
Comment (vijendarp) [inline pyswitch/snmp/mlx/base/utils.py]: is there a difference in ping command compared to SLX?

Comment (dpasupathi) [reply pyswitch/snmp/mlx/base/utils.py]: Yes.. both input and output differs

File: pyswitch/snmp/mlx/base/utils.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+        for key, value in cli_output.iteritems():
+            output_dict = {}
+            if ipv4_address.search(key):
+                ip = ipv4_address.search(key).group()
+            elif ipv6_address.search(key):
+                ip = ipv6_address.search(key).group()
+            value_list = value.splitlines()
+            success_rate = p_tx = p_rx = 0
+            p_loss = 100
+            for i, line in enumerate(value_list):
```
Comment (vijendarp) [inline pyswitch/snmp/mlx/base/utils.py]: Remove 'i' if it is not used

Comment (dpasupathi) [reply pyswitch/snmp/mlx/base/utils.py]: i is required as i am enumerating the list..

**Pull request merged**